src/components/Footer/index.js

```diff
@@ -11,21 +11,76 @@ import {
   SocialMediaLinks,
   FooterLinks,
   Copyright,
+  SocialMediaIcons,
+  FooterButtons,
 } from './StyledFooter'
 import { Col } from '../Grid'
+import WhatsAppImg from '../../assets/Images/whatsapp.svg'
+import PhoneImg from '../../assets/Images/phone.svg'
+import WebChatImg from '../../assets/Images/chat.svg'
+import { Heading } from '../Heading'
+import { Paragraph } from '../Paragraph'
+import { Inline } from '../Inline'
 
 const Footer = ({ as, direction, ...rest }) => {
   return (
     <FooterContainer fluid as="footer" {...rest}>
-      <ContactPanelRow debug noGutter>
-        <ContactPanelColumn sizes={{ xs: 12, md: 4 }} debug>
-          webchati
+      <ContactPanelRow>
+        <ContactPanelColumn sizes={{ xs: 12, md: 4 }} direction="row">
+          <WebChatImg width="60px" height="60px" alignSelf="start" />
+          <Heading level={4} textAlign="center">
+            Web Chat
+          </Heading>
+          <Paragraph textAlign="center" textSize="13px">
+            Got a question? Our advisers will point you in the right direction.
+          </Paragraph>
+          <Paragraph textAlign="center" textSize="13px" width="70%">
+            Monday to Friday, 8am to 6pm Saturday, 8am to 3pm Sunday and Bank
+            Holidays, closed
+          </Paragraph>
+          <FooterButtons text="Launch Chat" />
```

Comment:
Do we need a separate `<FooterButtons/>` component? Looks like it can use the default `<Button/>` styles.